controls/cis_rhel7.yml

```diff
@@ -2425,77 +2428,25 @@ controls:
     - accounts_no_uid_except_zero
 
   - id: 6.2.10
-    title: Ensure root PATH Integrity (Automated)
-    levels:
-    - l1_server
-    - l1_workstation
-    status: automated
-    rules:
-    - accounts_root_path_dirs_no_write
-    - root_path_no_dot
-
-  - id: 6.2.11
-    title: Ensure all users' home directories exist (Automated)
+    title: Ensure local interactive user home directories are configured (Automated)
     levels:
     - l1_server
     - l1_workstation
     status: automated
     rules:
     - accounts_user_interactive_home_directory_exists
-
-  - id: 6.2.12
-    title: Ensure users own their home directories (Automated)
-    levels:
-    - l1_server
-    - l1_workstation
-    status: automated
-    rules:
     - file_ownership_home_directories
-
-  - id: 6.2.13
-    title: Ensure users' home directories permissions are 750 or more restrictive (Automated)
-    levels:
-    - l1_server
-    - l1_workstation
-    status: automated
-    rules:
     - file_permissions_home_directories
 
-  - id: 6.2.14
-    title: Ensure users' dot files are not group or world writable (Automated)
-    levels:
-    - l1_server
-    - l1_workstation
-    status: automated
-    rules:
-      - accounts_user_dot_no_world_writable_programs
-
-  - id: 6.2.15
-    title: Ensure no users have .forward files (Automated)
-    levels:
-    - l1_server
-    - l1_workstation
-    status: automated
-    rules:
-      - no_forward_files
-
-  - id: 6.2.16
-    title: Ensure no users have .netrc files (Automated)
-    levels:
-    - l1_server
-    - l1_workstation
-    status: automated
-    notes: <-
-      The rule is checking only for existence of files, not for their permissions.
-    rules:
-    - no_netrc_files
-
-  - id: 6.2.17
-    title: Ensure no users have .rhosts files (Automated)
+  - id: 6.2.11
+    title: Ensure local interactive user dot files access is configured (Automated)
     levels:
     - l1_server
     - l1_workstation
-    status: automated
-    notes: The rule also removes /etc/hosts.equiv
+    status: partial
+    # TODO: add rule checking that .bash_history is mode 0600 or more restrictive
     rules:
+    - accounts_user_dot_no_world_writable_programs
+    - no_forward_files
```

Comment:
The polic states in the remediation part:
```
list .forward and .rhost files to be investigated and manually deleted
```
I think it would help to make these rules not remediate (not sure if it is possible from the top of my head), or move them to related_rules section.

Reply:
Yes, there is a small misalignment, but we keep the rules in many other profiles, so I think the remediation is harmless.

Reply:
OK... so please state it in the note.